Rename runcam device option to include caddx

ghostface opened the pull request:
So following the whole betaflight issue about implementing caddx protocol support in https://github.com/betaflight/betaflight/issues/6735 which ended up with "no new proprietary one-device protocols" I got into contact with caddx.

As the caddx and runcam protocols where not very different to begin with caddx agreed to align their protocol with runcams spec to make it compatible with the existing implementation.

Since the existing implementation now support both runcam and caddx cameras updated with the latest firmware I'd suggest to rename the option accordingly.

I attached the working caddx turtle firmware in my pullrequest if anyone wants to give it a try - I tested it sucessfully and both camera menu controls as well as the camera button work.

I talked about this with @mikeller on slack a few months back and he was forthcoming concerning this.

[caddx_turtle_firmware.zip](https://github.com/betaflight/betaflight-configurator/files/2873505/caddx_turtle_firmware.zip)

mikeller commented on locales/en/messages.json:
```diff
@@ -1332,7 +1332,7 @@
     },
     "portsFunction_RUNCAM_DEVICE_CONTROL": {
-        "message": "RunCam Device"
+        "message": "RunCam/Caddx Device"
```
Not sure about this - if more manufacturers start supporting this protocol then we'll have to change this again. I'd suggest we go for something like 'Camera device (RunCam protocol)'.

Thoughts? @azolyoung?

Also, note that our translations are done with an external tool (crowdin) by a group of volunteers. So it is not necessary to supply any translation except for English, the others will be translated in the tool, and the language files overwritten by the tool's output before every release.

walksnails commented:
the best choices is write : camera device ; no need include any company name

azolyoung commented:
Hello there,

Thanks @mikeller for @me, after seeing this PR and discussing it with our colleagues, we are confused about the request and the reason for this change. If you can change the RunCam Device to a RunCam/Caddx Device or Camera Device, can we change the name of the TBS SmartAudio/ IRC Tramp? Many hardware manufacturers also support these protocols on their hardware. According to the logic of ghostface, IRC Tramp can also be renamed, such as VTX Devcie (IRC/XXXX), but no one has asked for this. One reason is that this protocol is designed by ImmersionRC and was first applied to their products. The VTX can be controlled by Betaflight with this, which greatly facilitates the player. We respect and admire the efforts of ImmersionRC.

From the initial Runcam Split protocol (only for RunCam Split communication protocol), to Runcam Device Potocol, from blocked communication to current asynchronous, and unscheduledly follow-up the issues on Betaflight/iNav. We also have purchased a lot of equipment to solve the user's problems. We spent a lot of energy to do this.
We don't mind competitors enjoying the fruits of our labor. But if even the name of RunCam is to be removed or add the competitor's product name directly, we will be a bit disappointed.

mikeller commented:
@walksnails, @azolyoung: In my opinion 'camera device' is too vague - this will probably be confused with the camera control pin that is used to emulate a wired analogue control pad by many users. My preference is still 'Camera device (RunCam protocol)': This does not exclude any manufacturer / product, but is specific enough to state what protocol is used.

ghostface commented:
@mikeller gotcha concerning the translation tool - thank you for explaining that. I will change the pullrequest as soon as we decided on a route to proceed on.

I agree that "Camera Device" will probably be confused with the camera control pin so no good.
I guess "Camera Device (Runcam Protocol)" makes the most sense in this regard. 
However keeping it as "Runcam Device" suggests it only works with runcam devices which would be confusing.

I would also submit a change for the [betaflight wiki page](https://github.com/betaflight/betaflight/wiki/RunCam-Device-Protocol) indicating which cameras are supported to make it clear to the enduser.

walksnails commented:
@ghostface thanks for the reply ,we want to know ,how to make customer know caddx camera can use the protocol ?   and as us know, foxeer will not ask for change the name  , since we work with runcam togother, so we think just change the name to runcam/caddx protocol will be easy for both of us.

azolyoung commented:
Hi walksnails,

You just tell the users on the Ports tab and select the RunCam Device directly, and then it will work.

Whether it's Caddx earliest version, or the protocol you are using now，they are almost identical to RunCam Device Protocol. At the beginning, this protocol was designed and implemented by us and we keep optimized it. And for a long time, only us were using it. Compared to the number of times TBS SMARTAUDIO/IRC TRAMP is used by other manufacturers, the number of times RunCam Device protocol is used by other users is obviously small(as we know, only us). Why is it now required to modify the name of the entry just because another manufacturer has used this protocol? We know that the implementation of this protocol not only has our efforts (RunCam), there are also Martin, Mikeller and other developers in Betaflight who have given a lot of suggestions and corrections to the PR we submitted, we are grateful for this.But we strongly disagree with adding other brand names.

We are currently not sure whether foxxer, FXT or other camera manufacturers are interested in changing the name, and the PR itself is very strange. If the name can be changed, then other VTX manufacturers can also request to modify IRC TRAMP or TBS SMARTAUDIO?

ghostface commented:
How about this:

Camera (Runcam Protocol) and VTX (TBS Smartaudio) as well as VTX (IRC Tramp) to follow the same scheme.

I will amend the documentation concering the caddx camera support in the respective wiki pages as mentioned before.

ghostface commented:
I also updated the Wiki article for the device protocol to the best of my knowledge.

Someone from RunCam ( @azolyoung  ?) may want to update the list of supported cameras, I added the microswift3 and runcam racer 2 and checked if there are any others but I don't think there are yet.

I also added a link to the protocol from the initial pullrequest in betaflight, not sure if its the latest version. 
https://github.com/betaflight/betaflight/wiki/RunCam-Device-Protocol I rewrote some stuff to be more clear and comprehensive - please let me know if I got something wrong.
